lib/winapi/sync.c

```diff
@@ -580,3 +580,80 @@ BOOL ReleaseMutex (HANDLE hMutex)
     SetLastError(RtlNtStatusToDosError(status));
     return FALSE;
 }
+
```

Comment:
nit: Would expect commit title to be "winapi: Add Event functions", following scheme in https://github.com/XboxDev/nxdk/commit/69fbf00c959368989de458e4183dbddbd4b01b2d (note: "Add" vs "Implement" and "functions" vs "Functions")

Reply:
Ive changed the commit title